### PR TITLE
Add more DRBD resource metrics

### PR DESCRIPTION
**Pull request by kermat**
This pull request adds more of DRBD's metrics and the respective tests to the exporter.

### CODE REVIEW

File: drbd_metrics.go [hunk 1]
```diff
@@ -36,10 +46,20 @@ var (
...
+		"resources":            NewMetricDesc("drbd", "resources", "The DRBD resources; 1 line per name, per volume", []string{"resource", "role", "volume", "disk_state"}),
+		"written":              NewMetricDesc("drbd", "written", "KiB written to DRBD; 1 line per res, per volume", []string{"resource", "role", "volume", "disk_state"}),
+		"read":                 NewMetricDesc("drbd", "read", "KiB read from DRBD; 1 line per res, per volume", []string{"resource", "role", "volume", "disk_state"}),
+		"al_writes":            NewMetricDesc("drbd", "al_writes", "Writes to activity log; 1 line per res, per volume", []string{"resource", "role", "volume", "disk_state"}),
+		"bm_writes":            NewMetricDesc("drbd", "bm_writes", "Writes to bitmap; 1 line per res, per volume", []string{"resource", "role", "volume", "disk_state"}),
+		"upper_pending":        NewMetricDesc("drbd", "upper_pending", "Upper pending; 1 line per res, per volume", []string{"resource", "role", "volume", "disk_state"}),
+		"lower_pending":        NewMetricDesc("drbd", "lower_pending", "Lower pending; 1 line per res, per volume", []string{"resource", "role", "volume", "disk_state"}),
```
Comment (stefanotorresi) [inline drbd_metrics.go]: I'm not sure we need `disk_state` and `role` labels here, what do you think about removing them and leaving just `volume` and `resource`?

Comment (kermat) [reply drbd_metrics.go]: You're right, I'll drop those labels.

Comment (kermat) [reply drbd_metrics.go]: And, I'll update the docs while I'm at it :)

Comment (kermat) [reply drbd_metrics.go]: Done and pushed! Thanks @stefanotorresi

**Pull request merged**